packages/visx-react-spring/src/axis/AnimatedTicks.tsx

```diff
@@ -20,63 +20,56 @@ export default function AnimatedTicks<Scale extends AxisScale>({
   ticks,
   animationTrajectory,
 }: TicksRendererProps<Scale> & { animationTrajectory?: AnimationTrajectory }) {
-  const animatedTicks = useTransition(
-    ticks,
-    tick => `${tick.value}`,
-    useLineTransitionConfig({ scale, animateXOrY: horizontal ? 'x' : 'y', animationTrajectory }),
-  );
+  const animatedTicks = useTransition(ticks, {
+    ...useLineTransitionConfig({
+      scale,
+      animateXOrY: horizontal ? 'x' : 'y',
+      animationTrajectory,
+    }),
+    keys: (tick: { value: any }) => `${tick.value}`,
+  });
 
   return (
     // eslint-disable-next-line react/jsx-no-useless-fragment
     <>
-      {animatedTicks.map(
-        (
-          {
-            item,
-            key,
-            // @ts-ignore react-spring types only include CSSProperties
-            props: { fromX, toX, fromY, toY, opacity },
-          },
-          index,
-        ) => {
-          const tickLabelProps = allTickLabelProps[index] ?? allTickLabelProps[0] ?? {};
-          return item == null || key == null ? null : (
+      {animatedTicks(({ fromX, toX, fromY, toY, opacity }, item, { key }, index) => {
```

Comment:
nit: im getting a ts warning -> `Property 'fromX' does not exist on type '{}'.ts(2339)` for { fromX, toX, fromY, toY, opacity }, 
might be good to keep the @ts-ignore if no explicit type is defined
```
// @ts-ignore react-spring types only include CSSProperties
```

Reply:
It may be caused by the type inference issue of `9.1.2`.
After running yarn, it picks up`9.1.2` automatically instead of `9.0.0-rc.3`.
But I think it's not nice to stick to `9.0.0-rc.3` instead of using `^9.0.0-rc.3` in peer dependencies, so I think putting `@ts-ignore` here is indeed a good idea.

Reply:
looks like this is still broken in `9.2.0` so maybe `@ts-ignore` is the way to go 😢